run/idp-sql/test/e2e_test.py

```diff
@@ -179,7 +181,22 @@ def test_end_to_end(jwt_token: str, deployed_service: str) -> None:
     token = jwt_token
     service_url = deployed_service
 
+    retry_strategy = Retry(
+        total=3,
+        status_forcelist=[400, 401, 403, 500, 502, 503, 504],
+        allowed_methods=["GET", "POST"],
+        backoff_factor=3
+    )
+    retry_strategy_400 = Retry(
```

Comment:
Should this be named `retry_strategy_500`? Or is it 400 since 400s will not be retried :)?

Reply:
Good call. Updated.